### PR TITLE
RD-3660 Terraform modal improvements

### PR DESCRIPTION
**Pull request by kubama**
## Description

This PR addresses comments to #1710 that were not resolved there:
- Extended happy path cy test to cover generated blueprint deployment and installation
- Added length limit for input fields
- Simplified error message on ejs render failure
## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the UI Code Style.
- [x] I verified that all tests and checks have passed.
- [X] I verified if that change requires a documentation update.
- [X] I added proper labels to this PR.

## Tests
https://jenkins.cloudify.co/view/UI/job/Stage-UI-System-Test/1703/ (blueprint spec only)

 

## Documentation
N/A

### CODE REVIEW

File: widgets/blueprints/src/TerraformModal.tsx [hunk 1]
```diff
@@ -12,14 +12,29 @@ import type { Variable, Output } from '../../../backend/routes/Terraform.types';
...
 terraformVersionOptions[0].text = `${terraformVersionOptions[0].text} (${t('default')})`;
 
+export const inputMaxLength = 256;
+
+function LengthLimitedDynamicTableInput({ name, onChange, ...rest }: CustomConfigurationComponentProps<string>) {
+    return (
+        <Input
+            name={name}
+            onChange={(event, { value }) => onChange?.(event, { name, value: value as string })}
+            // eslint-disable-next-line react/jsx-props-no-spreading
```
Comment (leon-good-life) [inline widgets/blueprints/src/TerraformModal.tsx:30]: Just a comment,
Maybe we should remove the no-spreading rule from this eslint rule from the entire project.
I'm too getting eslint error when using {...props}.
I would like to raise it on our future UI meeting.

**Review by leon-good-life — COMMENTED** [review]
I checkout `RD-3660-e2e-test` branch on my computer I run `blueprints_spec`. One test didn't pass: 
Screen Shot 2021-12-22 at 12 08 55
Screen Shot 2021-12-22 at 12 10 19

Is it only on my machine?

File: widgets/blueprints/src/TerraformModal.tsx (hunk 1 above)
Comment (kubama) [reply widgets/blueprints/src/TerraformModal.tsx:30]: There is already Jira ticket for that: RD-1584

**Comment by kubama** [comment]
> Is it only on my machine?

This test will start working once it's on master. Please note the URL to the Terraform package used in the test:
https://github.com/cloudify-cosmo/cloudify-stage/raw/master/test/cypress/fixtures/terraform/local.zip
I wanted it to use a package from our repository, so I used reference to a file form the master branch, but it's not there yet. However, please note I tested it by using URL to development branch and it worked all fine.

**Review by leon-good-life — APPROVED** [review]

**Pull request merged**